packages/blockchain-wallet-v4/src/utils/btc.js

```diff
@@ -271,9 +272,13 @@ export const createXpubFromChildAndParent = (path, child, parent) => {
   let pathArray = bippath.fromString(path).toPathArray()
   let pkChild = compressPublicKey(Buffer.from(child.publicKey, 'hex'))
   let pkParent = compressPublicKey(Buffer.from(parent.publicKey, 'hex'))
-  let hdnode = fromPublicKey(pkChild, Buffer.from(child.chainCode, 'hex'))
-  hdnode.parentFingerprint = fingerprint(pkParent)
-  hdnode.depth = pathArray.length
-  hdnode.index = last(pathArray)
-  return hdnode.toBase58()
+  let hdnode = bip32.fromPublicKey(pkChild, Buffer.from(child.chainCode, 'hex'))
+  hdnode.__PARENT_FINGERPRINT = fingerprint(pkParent)
```

Comment:
It may not be possible to change, but accessing private members here.

Reply:
Noted on accessing private members here, the bip32 api does not expose setters.
https://github.com/bitcoinjs/bip32/blob/master/src/bip32.js

Reply:
Yeah, I figured, just wanted to make sure